sklearn/linear_model/_perceptron.py

```diff
@@ -97,20 +97,24 @@ class Perceptron(BaseSGDClassifier):
 
     Attributes
     ----------
+    classes_ : array of shape (n_classes,)
+        The unique classes labels.
+
     coef_ : array, shape = [1, n_features] if n_classes == 2 else [n_classes,\
             n_features]
         Weights assigned to the features.
 
     intercept_ : array, shape = [1] if n_classes == 2 else [n_classes]
         Constants in decision function.
 
+    loss_function_ : concrete LossFunction
+        The function that determines the loss, or difference between the \
```

Comment:
Mention that it is the perceptron loss (defined here https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/linear_model/_stochastic_gradient.py#L428)